tools/build/build.sh

```diff
@@ -20,17 +20,21 @@ set -e pipefail
 # Display commands
 set -x
 
+retry() {
+  "${@}" || "${@}" || "${@}" || exit $?
+}
+
 cd $(dirname $0)/../..
 BASE_DIR=$PWD
 
 ARTIFACTS_OUT="${BASE_DIR}/artifacts"
 mkdir -p "$ARTIFACTS_OUT"
 
-npm install
+retry npm install
 
 for version in 6.0.0 8.0.0 10.0.0 11.0.0 12.0.0
 do
-  ./node_modules/.bin/node-pre-gyp configure rebuild package \
+  retry ./node_modules/.bin/node-pre-gyp configure rebuild package \
```

Comment:
Does this do any network operations? If not, we should not retry it because we need to see the error.

Reply:
Yes, this does do network operations; specifically headers for different node versions are fetched. 

I'm adding the retry because this command failed with:

```
gyp ERR! stack Error: connect ETIMEDOUT 104.20.22.46:443
gyp ERR! stack     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1106:14)
gyp ERR! System Linux 4.15.16-041516-generic
```